Package.swift

```diff
@@ -25,6 +25,14 @@ let package = Package(
       name: "TokamakShim",
       targets: ["TokamakShim"]
     ),
+    .library(
+      name: "TokamakStatic",
```

Comment:
How do you feel about something like `TokamakHTML` or `TokamakStaticHTML`? Would that be more descriptive?

Reply:
I think `TokamakStaticHTML` is most descriptive.